help/deploy/complete-setup/configure-auth.md

```diff
@@ -3,24 +3,29 @@ title: "Configure authentication"
 sidebar_position: 1
 ---
 
-Out-of-the-box, Datagrok offers many authentication methods, including popular methods such as SSO and OAuth.
+Datagrok supports such authentication methods:
+
+* [Login-password](#login-password-authentication)
+* [Oauth](#oauth-authentication)
+* [OpenID](#openid-authentication)
+* [LDAP](#ldap-authentication)
 
 To accommodate any enterprise needs, we can develop a customer-specific integration.
 
-All methods can be enabled separately or combined.
+You can enable all methods separately or combined.
 
 [Learn more about authentification](../govern/authentication.md).
 
 ## Common options
 
-If you want to display something on the login window, set the 'Prompt' option
+To display something on the login window, set the 'Prompt' option
 ![Login prompt](login-prompt.png)
 
 Enable the 'Send Welcome Email' option to send welcome emails to new signed-up users.
 
 ## Login-password authentication
 
-It is the most basic method to authenticate users with Datagrok.
+This is a basic method to authenticate users with Datagrok.
 
 To configure login-password authentication for Datagrok:
 
```

Comment:
You should rewrite the instructions from a purpose-location-action perspective, and also use bold for UI elements like Sidebar, to make it easier for users to understand what to do.

The text also contains a lot of 'Datagrok', but it is not worth mentioning. Obviously, it is Datagrok settings, cause you are configuring auth in Datagrok.

Reply:
The 'email service' should not be a separate paragraph, it is important to mention and set a link to how-to, but it is not worth more than 1 sentence.

Reply:
Also, you can make the document more personable and easier to understand using 'You can' instead of 'Datagrok can integrate' and so on

Reply:
You should rewrite the instructions from a purpose-location-action perspective.
For example, instead of 
`Enable **Internal authentication** to use the login-password method` 
write 
`To use the login-password method, in the General section enable 'Internal authentication'`

Reply:
Same here, parameters are not UI elements, so using bold here is not a good idea. Italic would be better.

Reply:
The text also contains a lot of 'Datagrok,' which is redundant. And use bold for UI elements.
For example, instead of
`Go to the Datagrok Settings section **Users and Sessions**; this section contains all authentication settings.`
write
`On the left **Sidebar** go to **Settings** > **Users and Sessions**. Authentication settings page appears.`